uninstall.sh

```diff
@@ -137,10 +137,7 @@ uninstall_from_existing_repositories() {
 #   1 otherwise
 #####################################################
 is_running_in_git_repo_root() {
-    if ! git status >/dev/null 2>&1; then
-        return 1
-    fi
-
+    git rev-parse --is-bare-repository >/dev/null 2>&1 || return 1
```

Comment:
Same as above, this gives me `false` for a regular git repo.

Reply:
Yeah, but the point is, you only want to check if its a git repo :)
This function works because if its not a git repo (bare or not)
`git rev-parse --is-bare-repository` returns not `1`

Reply:
I have some issues with naming here.
It says `is_running_in_git_repo_root()` but AFAICT would work anywhere inside a git repo, not just the root.

Reply:
Also, the `git rev-parse --is-bare-repository` is still a bit interesting for me.
Can you please try if a simple `git rev-parse >/dev/null 2>&1 || return 1` would do what we want here and work for both bare repos and work trees?